modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,99 +82,562 @@ The following charts are available only if you are using Unified Agent and have
+
+Outbound - Response Time (99, 90, 75, and 50 percentile)::
+The average response time of the 99, 90, 75, and 50 percentiles of outbound requests, grouped by application in automatic intervals. The unit for these metrics is milliseconds (ms).
+
+Outbound - Failure Total Request Count::
+The sum of failed outbound requests, grouped by type of response in automatic intervals.
+
+Outbound - Ports::
+The sum of outbound requests, grouped by remote port in automatic intervals.
+
+// ports?
+
+Outbound - Operations::
+The sum of outbound requests, grouped by operation in automatic intervals.
+
+Outbound - Protocols::
+The sum of outbound requests, grouped by protocol in automatic intervals.
+
+Outbound - Endpoints::
+The sum of outbound requests, grouped by remote hosts (endpoints) in automatic intervals.
+
+[[flows-charts]]
+== Flows
+
+The following charts appear on the *Flows* dashboard:
+
+Inbound Response Time by Flow::
+The average inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average outbound response time, grouped by flow.
+
+Inbound Requests by Flow::
+The sum of inbound requests, grouped by flow.
+
+Outbound Requests by Flow::
+The sum of outbound requests, grouped by flow.
+
+Failures by Flow::
+The sum of failed inbound requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The sum of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The sum of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Inbound Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+
+Outbound Response Time by Flow::
+The average, minimum, and maximum outbound response time, grouped by flow.
+
+Inbound Requests by Flow::
+The sum of inbound requests, grouped by flow.
+
+Outbound Requests by Flow::
+The sum of outbound requests, grouped by flow.
+
+Failures by Flow::
+The sum of inbound failed requests, grouped by flow.
+//tbd?
+
+Slow Inbound Requests by Flow::
+The sum of inbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+Slow Outbound Requests by Flow::
+The sum of outbound requests whose response time is greater than 1000 ms, grouped by flow.
+
+[[connectors-charts]]
+== Connectors
+
+If you don't see the *Connectors* tab, verify that you have a Titanium subscription to Anypoint Platform and that the app you selected is a hybrid app deployed to a server, server group, or cluster. See xref:monitor-connectors.adoc[Monitor Connectors].
+
+The following charts appear on the *Connectors* dashboard:
+
+Total Connector Requests::
+The sum of outbound requests grouped by connector type in automatic intervals.
+
+Response Time by Connector::
+The average outbound response time grouped by connector type in automatic intervals.
+
+Salesforce Requests::
+The average number of Salesforce connector requests, grouped by operation in automatic intervals.
```

Comment:
What do you mean by "Salesforce connector"? Anypoint Connectors are Anypoint-Platform specific (MuleSoft), although there is a named "Anypoint Connector for Salesforce (Salesforce Connector)."
GLOBAL

Reply:
@jiang-wu @michael-lee-mulesoft Connector names?